lib/connectHapiServerToRedis.js

```diff
@@ -60,29 +83,31 @@ function connectHapiServerToRedis (server, cb) {
         
         // Example of unauth
         //cb( new Boom.unauthorized('Not authorized'));
+
+	      */
       }
     });
   });
-  // FIXIT - remove this once redis is integrated
-  cb(null, {status: "connected, but redis is not built out yet!"}); 
 
   // Configure the redis connection
   // This uses the hapi-redis npm package
-  // var redisOpts = {
-  //   connection: {
-  //     "host": "localhost", // Fixit. This is wrong.
-  //     "opts": {
-  //        "parser": "javascript",
-  //        "connect_timeout": 5000
-  //     }
-  //   }
-  // };
+   var redisOpts = {
+     connection: {
+       "host": "localhost", // Fixit. This is wrong.
+       // Later, if we want to use unix sockets instead, comment out the host line above and uncomment the socket line below.
+       //"socket": "/var/run/redis/redis.sock",
+       "opts": {
+          "parser": "javascript",
+          "connect_timeout": 5000
+       }
+     }
+   };
 
   // Register the redis plugin to the Hapi server
-  // server.register({
-  //   register: require('hapi-redis'),
-  //   options: redisOpts
-  // }, _handlePluginRegistration);
+  server.register({
+    register: require('hapi-redis'),
+    options: redisOpts
+  }, _handlePluginRegistration);
```

Comment:
Okay.  Looks good!  But what will happen when we merge this?  It seems like the app will fail to run unless it has a running redis instance, which as far as I know it doesn't yet...